### PR TITLE
Add changelog notes for DocBox 3.0

### PR DESCRIPTION
**Pull request by michaelborn**
* Refactors the changelog to the format suggested by Keep A Changelog
* Adds DocBox 3.0 release notes to changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
+# CHANGELOG
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+----
```
Comment (lmajano) [inline CHANGELOG.md:8]: Can you add the `----` separator between versions please. This is used to build latest changelogs

**Review by lmajano — CHANGES_REQUESTED** [review]
Just make sure you add the `----` separators between versions.

File: CHANGELOG.md (hunk 1 above)
Comment (michaelborn) [reply CHANGELOG.md:8]: Whoa! I did not realize that. Will do.

Comment (michaelborn) [reply CHANGELOG.md:8]: I added the `----` horizontal separators, Luis!

**Comment by michaelborn** [comment]
Updated branch per review.

**Pull request merged**